### DISCUSSION TITLE
I remade Friday Night Funkin' in Godot wanna know what ninjamuffin thinks of this shit lol

### DISCUSSION THREAD

**Discussion by rayanmargham** [FunkinCrew/Funkin#1842] [Show and tell]
i remade friday night funkin' into the godot game engine and shit wanna know what ninjamuffin thinks of this lol.

https://user-images.githubusercontent.com/69608259/138369317-5ad17d6c-b0de-4e5d-87c5-51857a5e2c4a.mp4

dont mind the watermark i was rushing cause i had to go sleep and crap (you can clearly see with the speed i was going through the menus with)

**Comment by libauror-a** [comment]
look good, keep up with the good work 👍

**Comment by KitOdin** [comment]
"Tankman's Cum Session"
Lmao XD

**Comment by Cheemsandfriends** [comment]
That's actually really good! GJ

**Comment by Techno274721** [comment]
HolY MoNgOlI hOwwww

**Comment by isakube** [comment]
"fard"

*intense Bushwhack*

**Comment by Reycko** [comment]
d

**Reply by libauror-a** [reply]
i

**Reply by KitOdin** [reply]
j

**Comment by Windows1Door2Daniel** [comment]
E

**Reply by KitOdin** [reply]
f

**Comment by Windows1Door2Daniel** [comment]
What are we writing?

**Reply by libauror-a** [reply]
idk man

**Reply by Windows1Door2Daniel** [reply]
> idk man

Agree

**Reply by KitOdin** [reply]
the alphabet dumass

**Reply by Windows1Door2Daniel** [reply]
> the alphabet dumass

Tf did you just said to me?

**Comment by ghost** [comment]
Funkin' Perfection
